src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("Search Results", () => {
+  const validProps = {
+    searchResults: ["test", "test"],
+  };
+
+  it("renders SearchResult component correctly", () => {
+    const { asFragment } = render(
+      <SearchResults results={validProps.results} />);
```

Comment:
```
      <SearchResults results={validProps.searchResults} />);
```
Your valid props has doesn't have a 'results' property but it does have a 'searchResults' property. Make sure you check the properties of your objects and console log them if you're not sure

You are testing against a snapshot that hasn't been updated. In other words, it's expecting the app to look the same as when you started working on it which will always fail. To fix this, just press 'u' after you run npm test. This will update the snapshot to include the changes that you've put in